packages/libs/lambda-at-edge/tests/routing/redirector.test.ts

```diff
@@ -25,18 +25,32 @@ describe("Redirector Tests", () => {
             destination: "/users/:id",
             statusCode: 307,
             regex: "^/old-users(?:/(\\d{1,}))$"
+          },
+          {
+            source: "/external",
+            destination: "https://example.com",
+            statusCode: 308,
+            regex: "^/external$"
+          },
+          {
+            source: "/invalid-destination",
+            destination: "ftp://example.com",
```

Comment:
Right now just supporting https:// and http:// protocol, even though this is a valid URL. Let me know if you are other thoughts